app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="library">Biblioteca de Sons</string>
+  <string name="about">Saiba mais</string>
+  <string name="report_issue">Informar sobre problema</string>
+  <string name="repeat_time_period">Período até repetir som</string>
+  <string name="volume">Volume do som</string>
+  <string name="play_pause">tocar ou pausar som</string>
+  <string name="open_drawer">Abrir menu de navegação</string>
+  <string name="close_drawer">Fechar menu de navegação</string>
+  <string name="notification_channel_default__name">Reprodução de audio</string>
+  <string name="notification_channel_default__description">Controles de audio</string>
+  <string name="pause">Pausar</string>
+  <string name="play">Tocar</string>
+  <string name="stop">Parar</string>
+  <string name="airplane_inflight">Avião voando</string>
+  <string name="airplane_seatbelt_beeps">Anúncio em avião</string>
```

Comment:
Google is translating this back to "Airplane Ad". Not sure why. Are you sure it says "Airplane seatbelt beeps"?

Reply:
It doesn't actually. I have never traveled by plane, and when I heard this sound I thought it sounded similar to a jingle that would play in a metro station before a message is transmitted; hence why I translated it as "announcement in an airplane".
The word "anúncio" can, however, be interpreted as "advertisement", so I think that's what happened here

Reply:
I'm not entirely happy with this translation either, so I'm open to suggestions!

Reply:
This message seems to refer to [this sound](https://www.youtube.com/watch?v=Y9WDcPwAQ0g), which is commonly turned on when starting a turbulence and when going to land the airplane, and turned off when in it is safe to walk around.

How about `Bipes de cinto de segurança de avião`? Is its length OK, @ashutoshgngwr ?

Reply:
@CebolaBros64 I am not sure either and I can't help. How about using a literal translation of "Airplane Seatbelt beeps"? :sweat_smile:

Reply:
@rffontenelle  Length can be a problem as it will make text go offscreen. Sound title is limited to single line only.

Reply:
Umm. If there is a simpler version of the translation then probably use that. Otherwise this is fine as well. We can make text go like marquee or something

Reply:
@CebolaBros64 how about "Bipes de cinto de avião" or "Cinto de segurança de avião"? Does it make sense to you?

Reply:
Yeah, sounds good. I think I'll go with the latter, since that sound is not exactly what I'd call a "beep"